test/unit/services/taxonomy/find_childest_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+require './lib/taxonomy'
+
+module Taxonomy
+  class FindChildestTest < ActiveSupport::TestCase
+    include EducationTaxonomyHelper
+
+    test '#taxons, returns childest taxons in a tree' do
+      stub_education_taxonomy
```

Comment:
`stub_education_taxonomy` is hiding important details of the test here, especially since we're working with content ids instead of titles.

It would be much easier to read what the test does if you had to explicitly tell it the basic structure of the taxonomy, even if you have to duplicate some code between tests.

what about something like this (very rough idea)

```
stub_taxonomy(
  content_item: "7c75c541-403f-4cb1-9b34-4ddde816a80d"
  children: {
    "7c75c541-403f-4cb1-9b34-4ddde816a80d" => ["904cfd73-2707-47b8-8754-5765ec5a5b68"],
    "904cfd73-2707-47b8-8754-5765ec5a5b68" => ["07fdd985-f3ec-4f4e-a316-3f4fd491bd64"]
  }
)
```

or making the stub class a builder for the stubs you actually create?

But if this isn't viable then I'd just inline the setup.